db/migrate/20200523215003_add_wallet_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddWalletToUser < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :wallet, :string, :default => 'bitsplit'
```

Comment:
Me da la impresión de que esto es como un `enum`? si es que si debería tener un `validates inclusion` en el modelo
Algo así
`validates :wallet, inclusion: { in: ['bitsplit', 'buda', 'alguna_otra_volaita']`
Es para asegurarse de que este restringido a solo los valores posibles.